src/main/java/org/fundacionjala/coding/Abner/NumberMap.java

```diff
@@ -0,0 +1,114 @@
+package org.fundacionjala.coding.Abner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class maps the number.
+ * Created by Samuel on 24/08/2017.
+ */
+public class NumberMap {
+
+    private static final int NUMBER_SIZE = 3;
+    private static final Map<String, Integer> MAP_NUMBER = new HashMap<>();
+
+    static {
+
+        MAP_NUMBER.put(String.format("%s%s%s", "   ",
+                "  |",
+                "  |"), 1);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                " _|",
+                "|_ "), 2);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                " _|",
+                " _|"), 3);
+
+        MAP_NUMBER.put(String.format("%s%s%s", "   ",
+                "|_|",
+                "  |"), 4);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "|_ ",
+                " _|"), 5);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "|_ ",
+                "|_|"), 6);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "  |",
+                "  |"), 7);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "|_|",
+                "|_|"), 8);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "|_|",
+                " _|"), 9);
+
+        MAP_NUMBER.put(String.format("%s%s%s", " _ ",
+                "| |",
+                "|_|"), 0);
+    }
+
+    /**
+     * this function changed the value string to int.
+     *
+     * @param stringCodeOne   this is the first string.
+     * @param stringCodeTwo   this is the second string.
+     * @param stringCodeThree this is the third string.
+     * @return number im string.
+     */
+    public String checkValue(String stringCodeOne, String stringCodeTwo, String stringCodeThree) {
+
+        StringBuilder numberMaps = new StringBuilder("");
```

Comment:
You don't need to initialize as empty string.
Remove the argument in the constructor.